src/main/java/org/prebid/server/activity/infrastructure/debug/ActivityInfrastructureDebug.java

```diff
@@ -56,6 +64,16 @@ public void emitActivityInvocationDefaultResult(boolean defaultResult) {
     }
 
     public void emitProcessedRule(Rule rule, Rule.Result result) {
+        if (rule instanceof AbstainPrivacyModule module) {
+            skippedModules.add(module.skippedModule());
+        } else if (rule instanceof AndRule andRule) {
+            CollectionUtils.emptyIfNull(andRule.rules()).stream()
+                    .filter(AbstainPrivacyModule.class::isInstance)
+                    .findFirst()
```

Comment:
We need to add all entries, there is a possibility for different modules be there